Updates to Flux `to()` (#3260)
influxdata/docs-v2: content/flux/v0.x/stdlib/experimental/to.md

| timestamp | measurement-name | field key | field value |

#### Data structure expected by experimental to()

> `experimental.to()` requires both the `_time` and `_measurement` columns

Reviewer: ```suggestion
`experimental.to()` requires both a `_time` and a `_measurement` column.
```

> Field keys and values are stored in single columns
> with the **field key** as the **column name**
> and the **field value** as the **column value**.
> 
> All columns in the group key (other than `_measurement`) are written as tags.
> All columns *not* in the group key (other than `_time`) are written as fields
> with the column name as the **field key** and the column values as the **field value**.

Reviewer: ```suggestion
All columns _in the group key_ (other than `_measurement`) are written as tags
with the column name as the **tag key** and the column value as the **tag value**.

All columns _**not** in the group key_ (other than `_time`) are written as fields
with the column name as the **field key** and the column value as the **field value**.
```

Only required when writing to a different organization or a remote host.
`orgID` and `org` are mutually exclusive.

{{% warn %}}

> It is not possible to write to from one InfluxDB Cloud organization to another

Reviewer: Technically you can write between organizations, you just can't with `experimental.to`.

```suggestion
`experimental.to()` cannot write to from one InfluxDB Cloud organization to another.
```
